app/Services/UpdateEmailService.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Brewmap\Services;
+
+use Brewmap\Eloquent\User;
+use Brewmap\Exceptions\User\NewEmailChangingException;
+use Brewmap\Notifications\EmailChangeNotification;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Notification;
+
+class UpdateEmailService
+{
+    /**
+     * Changes the user Email Address for a new one
+     */
+    public function sendNotifyForNewEmail(string $email): void
+    {
+        Notification::route("mail", $email)
```

Comment:
Try to remove the facade.

Reply:
Maybe `Illuminate\Notifications\ChannelManager` or `Illuminate\Notifications\Channels\MailChannel` will help

Reply:
Works with `Illuminate\Notifications\AnonymousNotifiable` :)
`